js_modules/dagit/packages/core/src/assets/LaunchAssetChoosePartitionsDialog.tsx

```diff
@@ -329,13 +329,12 @@ const LaunchAssetChoosePartitionsDialogBody: React.FC<Props> = ({
             <DimensionRangeWizard
               key={range.dimension.name}
               partitionKeys={range.dimension.partitionKeys}
-              partitionStateForKey={(dimensionKey) =>
-                mergedHealth.stateForSingleDimension(
+              health={{
```

Comment:
Unfortunately we use the same UI for both asset backfills / partition display and for op job backfills / partition display, and the op GraphQL API still returns one status object for every partition. We still need to support health data being provided in this way, so I turned this into a little union type.